tests/run/default/main.fmf

```diff
@@ -0,0 +1,6 @@
+summary: Check whether default plan works
+description:
+    If there is no plan detected a 'default plan' is used to allow
+    running tests and arbitrary commands without existing plan.
+    This should work both when no fmf metadata are found and when
+    there is a metadata tree but it does not contain any plan.
```

Comment:
Would it make sense to mention here what are the default steps of the default plan? Or link to them?

Reply:
Also it would be nice to document, how people should run with for example beakerlib ...

Now if users just run a beakerlib with default plan it will not work as expected

Reply:
Good point. Added a user story and examples.